### DISCUSSION TITLE
inpatient days and different rates based on day of stay

### DISCUSSION THREAD

**Discussion by jltang-kp** [CMSgov/price-transparency-guide#159] [Q&A] [answered]
How are orgs planning on displaying inpatient days beyond the first rate? I don't believe the schema supports tiered rates. We are thinking of using the description field in the in network object but would like opinions from other organizations. 

IE first DRG for days 1-3 is $300, days 4-5 is $200 and days after $100

**Comment by TimGallegos** [comment]
Due to the limitation of the MRF format, it suggests that all we can report is the inlier negotiated rate.

**Comment by jltang-kp** [comment]
@shaselton-usds  Following up on this topic, would it be sufficient for the regulation to report the first set of rates and omit following set of rates?

**Comment by shaselton-usds** [comment]
@jltang-kp could you create a negotiated rate object for each tier and the specifics of the tier are described in the `additional_information` section?

**Comment by jltang-kp** [comment, marked as answer]
@shaselton-usds Using the additional_information field was a possible idea we were thinking of to explain the difference in rate depending on the stay period ie rate $300 for day 1-3, days 4-5 is $200 etc. 

So to clarify, would it look like one negotiated rate object per unique rate for a given DRG and the additional_information field be the differentiator in these negotiated rate objects correct? Would this be sufficient to differentiate the rates?
